api/app/jobs/rdps_sfms.py

```diff
@@ -0,0 +1,181 @@
+"""
+A script that downloads RDPS temp, rh, precip and wind speed weather model data from NCEI NOAA HTTPS data server.
```

Comment:
This is coming from the env canada server, not NOAA correct?

Reply:
https://github.com/bcgov/wps/pull/3741/commits/dcc01d0f2a1f395ff42ae3275aafee8b26e4cb13